src/test/java/ladderGame/model/PlayerTest.java

```diff
@@ -0,0 +1,32 @@
+package ladderGame.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class PlayerTest {
+
+    @Test
+    @DisplayName("Line의 연결 여부에 따라 위치를 변경한다.")
+    void descendLine() {
+        Line line = new Line(
+                List.of(
+                        ConnectionStatus.CONNECTION,
+                        ConnectionStatus.DISCONNECTION,
+                        ConnectionStatus.CONNECTION
+                ));
+
+
+        Player player1 = new Player("pobi", 0);
+        Player player2 = new Player("crong", 1);
+
+        player1.move(line);
+        player2.move(line);
+
+        assertThat(player1.getPosition()).isEqualTo(1);
+        assertThat(player2.getPosition()).isEqualTo(0);
+    }
+}
```

Comment:
```suggestion
    @Test
    @DisplayName("Line의 연결 여부에 따라 위치를 변경한다.")
    void descendLine() {
        // given
        Line line = new Line(
                List.of(
                        ConnectionStatus.CONNECTION,
                        ConnectionStatus.DISCONNECTION,
                        ConnectionStatus.CONNECTION
                ));


        Player player1 = new Player("pobi", 0);
        Player player2 = new Player("crong", 1);

       // when
        player1.move(line);
        player2.move(line);

        // then
        assertThat(player1.getPosition()).isEqualTo(1);
        assertThat(player2.getPosition()).isEqualTo(0);
    }
}
```
given-when-then 을 명시해주면 다른 사람들이 더 쉽게 이해할 수 있을것 같아요!